os-server/src/main/java/app/openschool/course/api/dto/MentorCourseDto.java

```diff
@@ -0,0 +1,93 @@
+package app.openschool.course.api.dto;
+
+import java.util.Set;
+
+public class MentorCourseDto {
+
+  private String title;
+
+  private String description;
+
+  private Double rating;
+
+  private String categoryTitle;
+
+  private String difficultyTitle;
+
+  private String languageTitle;
+
```

Comment:
I think it will be better to rename the fileds categoryTitle, difficultyTitle, languageTitle to category, difficulty, language without adding title at the end

Reply:
renamed